arch/arm/src/armv8-m/arm_gen_nonsecfault.c

```diff
@@ -46,6 +46,17 @@
 #define OFFSET_R15             (6 * 4) /* R15 = PC */
 #define OFFSET_XPSR            (7 * 4) /* xPSR */
 
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+static uint32_t g_psp_ns;
+static uint32_t g_msp_ns;
```

Comment:
why need global data?

Reply:
we need to see last variable, from some debug tools, 
and this patch mainly for crash method